SampleUserControlLibrary/SampleUserControl.xaml.cs

```diff
@@ -98,11 +98,23 @@ public string SubscriptionKey
             set;
         }
 
+        public string SubscriptionEndpoint
+        {
+            get;
+            set;
+        }
+
+        public void SetSubscriptionPageEndpoint(string endpoint)
```

Comment:
It is not necessary to create a new function that doing the set action. You can just extend the `set` action above for `public string SubscriptionEndpoing`.